added note on required presence of requirements.txt (#125800)
Azure/azure-content: articles/app-service/configure-language-python.md

The App Service deployment engine automatically activates a virtual environment and runs `pip install -r requirements.txt` for you when you deploy a [Git repository](deploy-local-git.md), or when you deploy a [zip package](deploy-zip.md) [with build automation enabled](deploy-zip.md#enable-build-automation-for-zip-deploy).

> [!NOTE]

> > Currently App Service requires `requirements.txt` in your project's root directory, even if you're using modern Python packaging tools (i.e., `pyproject.toml`).

Reviewer: ```suggestion
> Currently App Service requires `requirements.txt` in your project's root directory, even if you're using modern Python packaging tools such as `pyproject.toml`.
```
